docs/zh_cn/04-supported-codebases/mmdet.md

```diff
@@ -210,7 +210,8 @@ cv2.imwrite('output_detection.png', img)
 |       [RepPoints](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/reppoints)       |    ObjectDetection    |      N      |    Y     |  N   |   ?   |    Y     |
 |            [DETR](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/detr)            |    ObjectDetection    |      Y      |    Y     |  N   |   ?   |    Y     |
 |       [CenterNet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/centernet)       |   Object Detection    |      Y      |    Y     |  N   |   ?   |    ?     |
-| [Cascade Mask R-CNN](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/cascade_rcnn) | InstanceSegmentation  |      Y      |    N     |  N   |   N   |    Y     |
+|       [RTMDet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet)             |   Object Detection    |      Y      |    Y     |  N   |   ?   |    ?     |
```

Comment:
The english md should be updated as well: `docs/en/04-supported-codebases/mmdet.md`

Reply:
I update English md in https://github.com/open-mmlab/mmdeploy/pull/1983, sry, as this is my first attempt at submitting a PR, I have split it into two separate PR submissions